test/MUXControlsReleaseTest/RuntimeComponentThatUsesMUX/TestUserControl1.idl

```diff
@@ -0,0 +1,8 @@
+﻿namespace RuntimeComponentThatUsesMUX
+{
+    [default_interface]
+    runtimeclass TestUserControl1 : Windows.UI.Xaml.Controls.UserControl
```

Comment:
>TestUserControl1 [](start = 17, length = 16)

I think that this is unused. Delete it if so.

Reply:
Ah, you're right - I was previously seeing if this would repro the issue, but it didn't.  I can remove it.